### ISSUE TITLE
Wishlist for extra features

### ISSUE THREAD

**Issue by Richardswe** [ansible-semaphore/semaphore#1038]
Hi i just wanted to say thanks for the work you guys put on Semaphore, i really like it.
I'd though have some extra features that would be really nice. I don't know how much work it is to add them
but again it would be appreciated :). 
Wish list:
1. Be able to move template tasks between the different tasks views. 
2. Add a static playbook instead of only relying on a GIT repository (a suggestion is that this could be stored locally on the node which runs Semaphore)
3. SSO authentication or Microsoft AD
4. Alerts and Success to Slack channel

**Comment by fiftin** [comment]
2 and 4 already available in develop branch and will be released soon.

**Comment by Richardswe** [comment]
@fiftin great!

**Comment by 95gabor** [comment]
Hi,

Is it possible to configure a task template with a git repository, but with using a static/local playbook file?

**Comment by Richardswe** [comment]
> Hi,
> 
> Is it possible to configure a task template with a git repository, but with using a static/local playbook file?

Yes that should be possible. I support of the use of static inventory but i suggest to use a git repository for playbooks. You can just place the inventory,playbooks etc locally on the Semaphore node.

**Comment by 95gabor** [comment]
> > Hi,
> > Is it possible to configure a task template with a git repository, but with using a static/local playbook file?
> 
> Yes that should be possible. I support of the use of static inventory but i suggest to use a git repository for playbooks. You can just place the inventory,playbooks etc locally on the Semaphore node.

Thanks!
I'm currently using Ansible Semaphore from a container and I do not want to attach volume to the container. Is it possible to use different repository or static file for the playbook and another for tasks? (the task repo is an open-source repository: ansible-openwrt and I don't want to fork it)

**Comment by Moep90** [comment]
`on Sep 30, 2022`
> 2. Add a static playbook instead of only relying on a GIT repository (a suggestion is that this could be stored locally on the node which runs Semaphore)

`on Oct 1, 2022`
> 2 and 4 already available in develop branch and will be released soon.

@fiftin  How can I do this? Could not find any documentation about it :-/ The last release was only three weeks ago.
